brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1869 - Button Block: Icon looks bigger on front end

### Description
Button Block: Icon looks bigger on front end

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

For old users - 

We can't manage this case. If the user adds HTML to our block, then we are not responsible for that. He/she should add text inside the button, and we have provided a link field where they can add a link.  To Resolve this for old users. Please check this video - [Screen Recording 10-10-2022 at 16.04](https://share.getcloudapp.com/9Zugz8pA)

Please Test backward compatibility from v1.25.6 and 2.0.13 users. If an Old user has added text to the button, it will not break. 

TC - 

1. Switch to branches v1.25.6 and v2.0.13
2. Drag and drop blocks which have buttons ( buttons, info-box, marketing button and CTA )
3. Add text with multiple variations like symbols, numbers etc.
4. Update the page and refresh front-end
5. Switch to the branch mentioned in this PR
6. Refresh the page and check whether the front end is working fine or not.
7. In the editor, please check for backward compatibility.

For new users - 

TC  - 
1. Switch to the branch mentioned in this PR
2. Drag and drop blocks which have buttons ( buttons, info-box, marketing button and CTA )
3.Paste links directly in the button text
4. Check it is pasted only string or link. 
5. Update the page 
6. Refresh the front end and check if it is working fine or not.

Testing Evidence - [Screen Recording 10-10-2022 at 15.59](https://share.getcloudapp.com/5zuP2Aov)

### Checklist:
